src/id5-api.js

```diff
@@ -42,35 +52,41 @@ ID5.init = function (options) {
       if (consent.isLocalStorageAllowed()) {
         utils.logInfo('Consent to access local storage and cookies is given');
 
-        if (!storedId || refreshNeeded) {
+        if (!storedResponse || !storedResponse.universal_uid || !storedResponse.signature || refreshNeeded) {
```

Comment:
nb is incremented even if no consent, maybe we should have two counters w and w/o ?

Reply:
for now, no need for two counters